appveyor.yml

```diff
@@ -1,33 +1,37 @@
 # version is set in tools\appveyor.psm1 - Invoke-AppVeyorInstall
+environment:
+  POWERSHELL_TELEMETRY_OPTOUT: 1
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1  # avoid expensive initialization of dotnet cli, see: http://donovanbrown.com/post/Stop-wasting-time-during-NET-Core-builds
+  matrix:
+  - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+    Purpose: BuildingAndPackaging
+  - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+    Purpose: BuildingAndTesting
 
-image: Visual Studio 2017
+# Stop all jobs in the matrix if any one them fail
+matrix:
+  fast_finish: true
 
-# cache version - netcoreapp.2.1-sdk.2.1.300
```

Comment:
Can you please leave this comment in place? It's needed to flush the nuget package cache when we update the sdk or runtime for the build.

Reply:
sorry, I didn't notice this.  Yes, please add this back

Reply:
Ok, added back